portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -231,6 +231,11 @@
 	<td>//*[@class="dropdown custom-views-actions"]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>HEADER_ROW</td>
+	<td>xpath=(//div[contains(@class, "dnd-tr")])[1]</td>
+	<td></td>
```

Comment:
For naming paths, generally I like to relate the name of the path with some attribute within the xpath. In this case, if there's some kind of header attribute or element that I can target that would be better in my opinion. But I understand what this means (first row of a table, which would be the header row).

Reply:
I'll keep that in mind for next time. I really didn't know how to name this path a clearer name.

Reply:
I'd prob suggest something like:
```
	<td>TABLE_ROW_1</td>
	<td>xpath=(//div[contains(@class, "dnd-tr")])[1]</td>
```

Other alternatives:
- if we think there'll be usecases to target other rows
```
	<td>TABLE_ROW_INDEX</td>
	<td>xpath=(//div[contains(@class, "dnd-tr")])[${index}]</td>
```
- or refactor this path, we try to have general paths in paths/pathlib that are shared between multiple components:
https://github.com/liferay/liferay-portal/commit/d4e8177b1c83d8e209ea1f51ba39dbd89cea7d4d